scripts/perf/docker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM debian:buster-20200514-slim
+
+WORKDIR /root
+
+RUN apt-get update \
+    && apt-get install -y \
+        git \
+        nvme-cli \
+        fio \
+	&& git clone \
+ 	 --recursive \
+	 --depth 1 \
+ 	 --shallow-submodules \
+ 	 --branch v20.04.1 \
+ 	 https://github.com/spdk/spdk \
+	&& /root/spdk/scripts/pkgdep.sh \
+	&& apt-get clean \
+    	&& rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```

Comment:
can we also build spdk?  Or is the build machine-dependent?

minor other tweak: indentation is a bit unaligned:

```
RUN apt-get .... \
 && apt-get install ...
      git ... \
 && git clone ..
```